src/code-components/RichTextArea/RichTextArea.tsx

```diff
@@ -40,40 +43,35 @@ export function RichTextArea({
   wrapperClassName,
   ariaLabel,
   ariaLabeledby,
-  role,
+  fallback,
 }: RichTextAreaProps) {
-  const Editor = useMemo(() => {
-    return dynamic(() => import("./Editor").then((module) => module.Editor), {
-      ssr: false,
-    });
-  }, []);
-  const quillRef = useRef<Quill | null>(null);
-
   const formattedToolbar = useMemo(
     () => formatDefaultToolbarConfigs(toolbar),
     [JSON.stringify(toolbar)],
```

Comment:
Cant it be simply?

```suggestion
    [toolbar],
```

Won't react's dependency comparison handle the object reference?

Reply:
That depends whether plasmic gives the same reference or a new object on each render.